docs/tables/oci_application_migration_source.md

```diff
@@ -0,0 +1,152 @@
+---
+title: "Steampipe Table: oci_application_migration_source - Query OCI Application Migration Sources using SQL"
+description: "Allows users to query data from Oracle Cloud Infrastructure's Application Migration Sources."
+---
+
+# Table: oci_application_migration_source - Query OCI Application Migration Sources using SQL
+
+**Deprecated. Use [azuread_group](https://hub.steampipe.io/plugins/turbot/azuread/tables/azuread_group) instead.**
```

Comment:
Please update the reference to use the corresponding `oci_*` table instead of `azuread_*` table.